### PR TITLE
Adds spec and test data for multiple sections op one section.

### PR DESCRIPTION
**Pull request by anupcowkur**

### CODE REVIEW

**Comment by anupcowkur** [comment]
Ready for review.

File: input_multiple_sections_op_one_section.json [hunk 1]
```diff
@@ -0,0 +1,116 @@
+{
```
Comment (adsantos) [inline input_multiple_sections_op_one_section.json]: Is this file the same as other input files? If yes, why not reusing it? The same applies on the other PR.

Comment (adsantos) [inline input_multiple_sections_op_one_section.json]: If the name of the file bothers you, just name it something generic.

File: spec_multiple_sections_op_one_section.json [hunk 2]
```diff
@@ -0,0 +1,103 @@
+{
+    "queries": [
+        {
+            "name": "multiple sections op one section",
+            "behaviour": [
+                "Multiple sections are generated only when using the ',', operator.",
+                "When an operator is applied to multiple sections as one operand and a single section as the other operand,",
+                "The result of the operation is multiple sections that are generated using the rules of mathematical logic."
```
Comment (adsantos) [inline spec_multiple_sections_op_one_section.json]: Same comment as on the previous PR - a more descriptive behaviour would help, I think.

File: input_multiple_sections_op_one_section.json (hunk 1 above)
Comment (anupcowkur) [inline input_multiple_sections_op_one_section.json]: the tasks have the same names but there are subtle differences in their properties.

Comment (anupcowkur) [inline input_multiple_sections_op_one_section.json]: @adsantos I've consolidated them now. `spec_operators`, `spec_multiple_sections_op_one_section` and `spec_multiple_sections_op_multiple_sections` all use the same `input_operators` test file.

**Pull request merged**